docs/data-sources/build.md

```diff
@@ -25,6 +27,8 @@ data "kustomization_build" "test" {
 #### Child attributes
 
 - `load_restrictor` - setting this to `"none"` disables load restrictions
+- `enable_helm` - setting this to `true` enabled helm templating support
+- `helm_path` - set this to the path to find the helm executable (default: `"helmV3"`)
```

Comment:
The binary is called `helmV3` by upstream?

Reply:
```suggestion
- `helm_path` - set this to the path to the binary (defaults to: `helm`)
```

Reply:
it is indeed called `helmV3` by upstream. I'll correct the rest however.